fastcdc/fastcdc.h

```diff
@@ -57,53 +60,53 @@ using ChunkFoundHandler = std::function<void(const uint8_t* data, size_t len)>;
 // Implements a very fast content-defined chunking algorithm.
 //
 // FastCDC [1] identifies chunk boundaries based on a simple yet efficient
```

Comment:
If we go with this implementation, it doesn't make much sense anymore to call it fastcdc, I guess.

Reply:
Yeah, probably correct. I wanted to minimize the changes for now. I think git can also get a bit confused merging things after directory renames (or is that old news?).

Reply:
It's a pretty disruptive change to rename it... it's all over the codebase.

Personally I'm OK with leaving it as it is. Although it's not the same as the original fastcdc, the research and changes I did started with investigating it, and the changes are well documented in comments.